### PR TITLE
[DOC-12362]: Feedback on Swap Space and Kernel Swappiness | RHEL8 Known Issue

### PR DESCRIPTION
**Pull request by RayOffiah**
Added a sidebar and a link explaining the problem and the resolution.

https://issues.couchbase.com/browse/DOC-12362

### CODE REVIEW

File: modules/install/pages/install-swap-space.adoc [hunk 1]
```diff
@@ -51,6 +51,7 @@ If the output doesn't match the <<recommended-swappiness-settings,recommended se
 
 . Change the swappiness setting.
 +
+[upperroman]
```
Comment (osfameron) [inline modules/install/pages/install-swap-space.adoc]: This is the bulleted list style?
Is this part of our standard tone? I don't find `upperroman` elsewhere in docs-server

**Review by osfameron — APPROVED** [review]
Looks good (one comment about `upperroman` inline)

File: modules/install/pages/install-swap-space.adoc (hunk 1 above)
Comment (RayOffiah) [reply modules/install/pages/install-swap-space.adoc]: No, that's Roman numbers for an ordered sublist. 

I'd better check the style guide and see what the form is for sublists. 

…
…

https://docs.couchbase.com/styleguide/ordered-list.html
Not too much comment around that area.

I'll drop a note to Sarah.

Comment (RayOffiah) [reply modules/install/pages/install-swap-space.adoc]: Sorted. Had a chat with Sarah, and I've left as the Antora default.

**Pull request merged**